.github/workflows/xds-sync-apis.yml

```
@@ -21,3 +21,5 @@ jobs:
     uses: ./.github/workflows/xds-apply-updates.yml
     with:
       target_version: ${{ needs.envoy-versions.outputs.target_version }}
+    secrets:
+      ARMERIAN_PAT: ${{ secrets.ARMERIAN_PAT }}
```

actionlint (1.7.12)

[error] secret "ARMERIAN_PAT" is not defined in "./.github/workflows/xds-apply-updates.yml" reusable workflow. no secret is defined

(workflow-call)